### PR TITLE
SDIT-2337: 💄 Tidy up pdf header and footer

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: server/config.ts [hunk 1]
```diff
@@ -89,13 +89,6 @@ export default {
     },
     gotenberg: {
       url: get('GOTENBERG_API_URL', 'http://localhost:3001', requiredInProduction),
-      pdfMargins: {
```
Comment (petergphillips) [inline server/config.ts:92]: this ended up being annoying, so moved down into the gotenberg client itself

File: server/routes/print/printController.test.ts [hunk 2]
```diff
@@ -158,15 +157,10 @@ describe('Print controller', () => {
 
       expect(res.renderPdf).toHaveBeenCalledWith(
         'pages/pdf',
-        { ...detail },
```
Comment (petergphillips) [inline server/routes/print/printController.test.ts:161]: don't need 3 different data sections, can have one single one and use for all

File: server/views/pages/pdfHeader.njk [hunk 3]
```diff
@@ -1,38 +1,55 @@
...
+  <title>Unused</title>
 </head>
 <body>
-    <div class='adjudication-pdf-header'>
-        <h1>Conduct report</h1>
-        <h2><span class='adjudication-pdf-header-bold'>Charge number:</span> <span class="adjudication-pdf-header-normal">{{ chargeNumber }}</span></h2>
-    </div>
+<div>
+  <div class='dps-logo'>
+    <svg width="100" height="100" viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (petergphillips) [inline server/views/pages/pdfHeader.njk:27]: copied from `dps-logo.svg` in DPS.  Needs to be included here separately as gotenberg headers can't read images

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**